pkg/dns/server.go

```diff
@@ -81,6 +81,7 @@ func (d *SimpleDNSServer) Start(stop <-chan struct{}) error {
 
 	errChan := make(chan error)
 	go func() {
+		defer close(errChan)
```

Comment:
Why are we closing here the channel? We are not doing it in other servers.

Reply:
It is a normal practice we do.